editor-service/src/main/resources/dsm/elementsTypes_en.json

```diff
@@ -0,0 +1,385 @@
+                            "value": "trikWaitGamepadDisconnectBlock"
+                        },
+                        {
+                            "key": "trikWaitGamepadWheelBlock",
+                            "value": "trikWaitGamepadWheelBlock"
+                        },
+                        {
+                            "key": "trikWaitPadPressBlock",
+                            "value": "trikWaitPadPressBlock"
+                        },
+                        {
+                            "key": "variableInitBlock",
+                            "value": "variableInitBlock"
+                        },
+                        {
+                            "key": "waitForAccelerometerBlock",
+                            "value": "waitForAccelerometerBlock"
+                        },
+                        {
+                            "key": "waitForButtonBlock",
+                            "value": "waitForButtonBlock"
+                        },
+                        {
+                            "key": "waitForColorBlock",
+                            "value": "waitForColorBlock"
+                        },
+                        {
+                            "key": "waitForColorIntensityBlock",
+                            "value": "waitForColorIntensityBlock"
+                        },
+                        {
+                            "key": "waitForEncoderBlock",
+                            "value": "waitForEncoderBlock"
+                        },
+                        {
+                            "key": "waitForGyroscopeBlock",
+                            "value": "waitForGyroscopeBlock"
+                        },
+                        {
+                            "key": "waitForLightBlock",
+                            "value": "waitForLightBlock"
+                        },
+                        {
+                            "key": "waitForMessageBlock",
+                            "value": "waitForMessageBlock"
+                        },
+                        {
+                            "key": "waitForMotionBlock",
+                            "value": "waitForMotionBlock"
+                        },
+                        {
+                            "key": "waitForSonarDistanceBlock",
+                            "value": "waitForSonarDistanceBlock"
+                        },
+                        {
+                            "key": "waitForSoundSensorBlock",
+                            "value": "waitForSoundSensorBlock"
+                        },
+                        {
+                            "key": "waitForTouchSensorBlock",
+                            "value": "waitForTouchSensorBlock"
+                        },
+                        {
+                            "key": "writeToFileBlock",
+                            "value": "writeToFileBlock"
+                        }
+                    ]
```

Comment:
since we have more than one editor now, we definitely need a wiki page explaining the format. may be even use something line http://json-schema.org/?